### DISCUSSION TITLE
cant apply font from google font url

### DISCUSSION THREAD

**Discussion by GCoboFacePhi** [chakra-ui/panda#3006] [General]
### Description

I am creating an internal component library, I am including the font globally using the google fonts url, in the network tab the font loads without problem, in the generated css the font-face is applied well, but when applying the font it is not done correctly.

```ts 
//panda.config.ts
globalFontface: {
    Poppins: [
      {
        src: 'url(https://fonts.googleapis.com/css2?family=Poppins&display=swap)',
        fontWeight: '400',
        fontStyle: 'normal',
        fontDisplay: 'swap',
      },
      {
        src: 'url(https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap)',
        fontWeight: '600',
        fontStyle: 'normal',
        fontDisplay: 'swap',
      },
    ],
  },
  
  ```
  
  ```ts
  ///css generated output
    @font-face {
    font-family: Poppins;
    src: url(https://fonts.googleapis.com/css2?family=Poppins&display=swap);
    font-weight: 400;
    font-style: normal;
    font-display: swap;
  }
  @font-face {
    font-family: Poppins;
    src: url(https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap);
    font-weight: 600;
    font-style: normal;
    font-display: swap;
  }
  ```
  
  
![image](https://github.com/user-attachments/assets/76347469-264d-47dc-845b-e3b7535b2a8e)

![image](https://github.com/user-attachments/assets/6c3f8e01-0811-4a72-91d5-dd7aae8dfa71)

  
  
  

### Link to Reproduction

https://play.panda-css.com/kPWg1JltxP

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### JS Framework

React

### Panda CSS Version

0.48.0

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by anubra266** [comment]
@GCoboFacePhi, `globalFontFace` is for generating `@font-face` which is not the way to use google fonts. The way ti use google fonts is to `@import` that url:
Like:

https://play.panda-css.com/kPWg1JltxP/CoyWzYzm6A?tab=css

For easier ways to manage your fonts, you can check out the [font docs](https://panda-css.com/docs/guides/fonts)
